bms.py

```diff
@@ -0,0 +1,45 @@
+from math import isnan
+from bmsConstants import bms_param_thresholds 
+
+def get_param_min_limit(param_name):
+    return bms_param_thresholds[param_name]['min']
+
+def get_param_max_limit(param_name):
+    return bms_param_thresholds[param_name]['max']
+
+
+def validate_bms_param_name(bms_parameter):
+    for param_name in bms_parameter.keys():
+        if param_name not in bms_param_thresholds:
+            return 'ParamNameUnknown'
+    return 'OK'
+
+def validate_bms_param_value(bms_parameter):

```

Comment:
This function is tested but not used within this code. Is it meant to be used by a consumer of this code?